ee/codegen/src/types/vellum.ts

```diff
@@ -942,7 +942,7 @@ export type MCPServerFunctionArgs = {
   type: "MCP_SERVER";
   name: WorkflowValueDescriptor;
   url: WorkflowValueDescriptor;
-  authorization_type: "BEARER_TOKEN" | "API_KEY";
+  authorization_type: WorkflowValueDescriptor;
```

Comment:
[q] why do we need to workflow value descriptor this guy too? it's unlikely that we make this dynamic